### PR TITLE
[trivial] lint: skip generated files

### PR DESCRIPTION
**Pull request by halseth**
Would fail on files created by mobile build tools, since they redeclare certain method names.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ run:
   # worker.
   concurrency: 1
 
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 5m
```
Comment (wpaulino) [inline .golangci.yml]: What's the rationale for this change? Would be nice to include it in the commit message.

Comment (halseth) [reply .golangci.yml]: just to try to make travis pass

Comment (halseth) [reply .golangci.yml]: Removed

**Comment by halseth** [comment]
Travis should be fixed now. Also added verbose mode to the linter to get more insight into what it is doing.
